wagtail/wagtailcore/tests/test_blocks.py

```diff
@@ -608,6 +610,34 @@ def test_invalid_searchable_content(self):
         ])
         self.assertEqual(block.get_searchable_content('three'), [])
 
+    def test_searchable_content_with_lazy_translation(self):
+        block = blocks.ChoiceBlock(choices=[
+            ('choice-1', _("Choice 1")),
```

Comment:
Might be a good idea to use a different alias for ugettext_lazy to prevent these strings from ending up on transifex and wasting translator's time